{"tensorflow.python.ops.gen_nn_ops.max_pool": ["tensorflow.python.eager.execute.make_str", "tensorflow.python.eager.execute.record_gradient", "<builtin>.isinstance", "<builtin>.TypeError", "tensorflow.python.eager.execute.make_int", "tensorflow.python.ops.gen_nn_ops.max_pool_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.variable_scope.get_variable": ["tensorflow.python.ops.variable_scope._get_default_variable_store", "tensorflow.python.ops.variable_scope.get_variable_scope"], "tensorflow.python.ops.nn_impl.sigmoid_cross_entropy_with_logits": ["tensorflow.python.ops.nn_ops._ensure_xent_args", "tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.ops.array_ops.zeros_like", "tensorflow.python.ops.array_ops.where", "<builtin>.ValueError"], "tensorflow.python.ops.gen_nn_ops.relu": ["tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_nn_ops.relu_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.training.adam.AdamOptimizer.__init__": ["<builtin>.super"], "tensorflow.python.framework.ops.name_scope.__init__": ["tensorflow.python.eager.context.context"], "tensorflow.python.ops.gen_nn_ops.bias_add": ["tensorflow.python.eager.execute.make_str", "tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_nn_ops.bias_add_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.client.session.Session.__init__": ["tensorflow.python.client.session.BaseSession.__init__", "<builtin>.super"], "tensorflow.python.training.saver.Saver.__init__": ["<builtin>.ValueError", "tensorflow.python.eager.context.executing_eagerly", "<builtin>.RuntimeError", "time.time", "tensorflow.python.training.saver.Saver.build", "tensorflow.python.training.saver.Saver._check_saver_def"], "tensorflow.python.ops.array_ops.placeholder": ["tensorflow.python.eager.context.executing_eagerly", "<builtin>.RuntimeError", "tensorflow.python.ops.gen_array_ops.placeholder"], "tensorflow.python.ops.math_ops.argmax": ["tensorflow.python.util.deprecation.deprecated_argument_lookup", "tensorflow.python.ops.gen_math_ops.arg_max"], "tensorflow.python.ops.gen_math_ops.cast": ["tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.make_bool", "tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_math_ops.cast_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.gen_math_ops.add": ["tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_math_ops.add_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.nn_ops.dropout": ["tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.ops.nn_ops._get_noise_shape", "tensorflow.python.ops.random_ops.random_uniform", "<builtin>.ValueError", "<builtin>.isinstance", "tensorflow.python.ops.math_ops.div", "tensorflow.python.framework.tensor_shape.scalar", "tensorflow.python.framework.tensor_util.constant_value"], "tensorflow.python.ops.gen_math_ops.equal": ["tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_math_ops.equal_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.framework.ops.Graph.__init__": ["threading.RLock", "tensorflow.python.util.lock_util.GroupLock.__init__", "<builtin>.dict", "threading.local", "tensorflow.python.framework.traceable_stack.TraceableStack.__init__", "collections.OrderedDict", "<builtin>.set", "tensorflow.python.framework.ops.uid", "tensorflow.python.framework.op_def_registry.get_registered_ops", "tensorflow.python.framework.c_api_util.ScopedTFGraph.__init__"], "tensorflow.python.ops.variables.global_variables_initializer": ["tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.ops.variables.variables_initializer", "tensorflow.python.ops.variables.global_variables"], "tensorflow.python.ops.math_ops.matmul": ["tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.eager.context.executing_eagerly", "<builtin>.ValueError", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.ops.gen_math_ops.batch_mat_mul", "tensorflow.python.ops.math_ops.conj", "tensorflow.python.ops.gen_math_ops.mat_mul", "<builtin>.isinstance", "tensorflow.python.ops.math_ops.cast", "<builtin>.len"], "tensorflow.python.summary.writer.writer.FileWriter.__init__": ["tensorflow.python.eager.context.executing_eagerly", "<builtin>.RuntimeError", "tensorflow.python.summary.writer.event_file_writer_v2.EventFileWriterV2.__init__", "tensorflow.python.summary.writer.event_file_writer.EventFileWriter.__init__", "tensorflow.python.summary.writer.writer.SummaryToEventTransformer.__init__", "<builtin>.super"], "tensorflow.python.ops.gen_array_ops.reshape": ["tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_array_ops.reshape_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.math_ops.reduce_min": ["tensorflow.python.util.deprecation.deprecated_argument_lookup", "tensorflow.python.ops.math_ops._may_reduce_to_scalar", "tensorflow.python.ops.gen_math_ops._min", "tensorflow.python.ops.math_ops._ReductionDims"], "tensorflow.python.ops.random_ops.random_normal": ["tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.ops.random_ops._ShapeTensor", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.framework.random_seed.get_seed", "tensorflow.python.ops.gen_random_ops.random_standard_normal"], "tensorflow.python.ops.gen_nn_ops.conv2d": ["tensorflow.python.eager.execute.make_str", "tensorflow.python.eager.execute.make_bool", "tensorflow.python.eager.execute.record_gradient", "<builtin>.isinstance", "<builtin>.TypeError", "tensorflow.python.eager.execute.make_int", "tensorflow.python.ops.gen_nn_ops.conv2d_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.contrib.layers.python.layers.initializers.xavier_initializer": ["tensorflow.contrib.layers.python.layers.initializers.variance_scaling_initializer"], "tensorflow.python.ops.math_ops.reduce_mean": ["tensorflow.python.util.deprecation.deprecated_argument_lookup", "tensorflow.python.ops.math_ops._may_reduce_to_scalar", "tensorflow.python.ops.gen_math_ops.mean", "tensorflow.python.ops.math_ops._ReductionDims"], "tensorflow.python.eager.execute.make_str": ["<builtin>.isinstance", "<builtin>.TypeError", "<builtin>.repr", "tensorflow.python.util.compat.as_bytes"], "tensorflow.python.eager.execute.make_int": ["<builtin>.isinstance", "<builtin>.TypeError", "<builtin>.repr", "<builtin>.int"], "tensorflow.python.ops.gen_nn_ops.max_pool_eager_fallback": ["tensorflow.python.eager.context.context", "<builtin>.isinstance", "<builtin>.TypeError", "tensorflow.python.eager.execute.make_int", "tensorflow.python.eager.execute.make_str", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.variable_scope._get_default_variable_store": ["tensorflow.python.framework.ops.get_collection", "tensorflow.python.ops.variable_scope._VariableStore.__init__", "tensorflow.python.framework.ops.add_to_collection"], "tensorflow.python.ops.variable_scope.get_variable_scope": ["tensorflow.python.ops.variable_scope.get_variable_scope_store"], "tensorflow.python.ops.nn_ops._ensure_xent_args": ["<builtin>.ValueError"], "tensorflow.python.framework.ops.convert_to_tensor": ["tensorflow.python.framework.ops.internal_convert_to_tensor"], "tensorflow.python.ops.array_ops.zeros_like": ["tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.ops.array_ops.zeros", "tensorflow.python.ops.gen_array_ops.zeros_like", "tensorflow.python.framework.ops.device", "tensorflow.python.ops.array_ops.shape_internal"], "tensorflow.python.ops.array_ops.where": ["tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.ops.gen_array_ops.where", "tensorflow.python.ops.gen_math_ops.select", "<builtin>.ValueError"], "tensorflow.python.ops.gen_nn_ops.relu_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.eager.context.context": ["tensorflow.python.eager.context._initialize_context"], "tensorflow.python.ops.gen_nn_ops.bias_add_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_str", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.client.session.BaseSession.__init__": ["tensorflow.python.framework.ops.get_default_graph", "<builtin>.isinstance", "<builtin>.TypeError", "<builtin>.type", "threading.Lock", "tensorflow.python.util.compat.as_bytes"], "tensorflow.python.eager.context.executing_eagerly": ["tensorflow.python.eager.context.context"], "tensorflow.python.training.saver.Saver.build": ["tensorflow.python.eager.context.executing_eagerly", "<builtin>.RuntimeError", "tensorflow.python.training.saver.Saver._build"], "tensorflow.python.training.saver.Saver._check_saver_def": ["<builtin>.isinstance", "<builtin>.ValueError", "tensorflow.python.eager.context.executing_eagerly", "<builtin>.str"], "tensorflow.python.ops.gen_array_ops.placeholder": ["tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.make_shape", "tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_array_ops.placeholder_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.util.deprecation.deprecated_argument_lookup": ["<builtin>.ValueError"], "tensorflow.python.ops.gen_math_ops.arg_max": ["tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_math_ops.arg_max_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.eager.execute.make_type": ["tensorflow.python.framework.dtypes.as_dtype", "<builtin>.TypeError", "<builtin>.repr"], "tensorflow.python.eager.execute.make_bool": ["<builtin>.isinstance", "<builtin>.TypeError", "<builtin>.repr"], "tensorflow.python.ops.gen_math_ops.cast_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.make_bool", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_math_ops.add_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.nn_ops._get_noise_shape": ["tensorflow.python.ops.array_ops.shape", "tensorflow.python.framework.tensor_shape.as_shape", "<builtin>.enumerate", "tensorflow.python.framework.tensor_shape.TensorShape.__init__", "<builtin>.len"], "tensorflow.python.ops.random_ops.random_uniform": ["tensorflow.python.framework.dtypes.as_dtype", "<builtin>.ValueError", "tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.ops.random_ops._ShapeTensor", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.framework.random_seed.get_seed", "tensorflow.python.ops.gen_random_ops.random_uniform_int", "tensorflow.python.ops.gen_random_ops.random_uniform"], "tensorflow.python.ops.math_ops.div": ["tensorflow.python.ops.math_ops._div_python2"], "tensorflow.python.framework.tensor_util.constant_value": ["<builtin>.isinstance", "tensorflow.python.framework.tensor_util._ConstantValue"], "tensorflow.python.ops.gen_math_ops.equal_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.util.lock_util.GroupLock.__init__": ["<builtin>.ValueError", "threading.Condition", "threading.Lock"], "tensorflow.python.ops.variables.variables_initializer": ["tensorflow.python.ops.control_flow_ops.group", "tensorflow.python.eager.context.executing_eagerly"], "tensorflow.python.ops.variables.global_variables": ["tensorflow.python.framework.ops.get_collection"], "tensorflow.python.ops.gen_math_ops.batch_mat_mul": ["tensorflow.python.eager.execute.make_bool", "tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_math_ops.batch_mat_mul_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.math_ops.conj": ["<builtin>.isinstance", "tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.ops.gen_math_ops.conj", "<builtin>.TypeError"], "tensorflow.python.ops.gen_math_ops.mat_mul": ["tensorflow.python.eager.execute.make_bool", "tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_math_ops.mat_mul_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.math_ops.cast": ["tensorflow.python.framework.dtypes.as_dtype", "<builtin>.isinstance", "tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.ops.math_ops.cast", "tensorflow.python.framework.sparse_tensor.SparseTensor.__init__", "tensorflow.python.platform.tf_logging.warn", "tensorflow.python.framework.ops.IndexedSlices.__init__", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.ops.gen_math_ops.cast"], "tensorflow.python.summary.writer.event_file_writer_v2.EventFileWriterV2.__init__": ["tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.ops.summary_ops_v2.create_file_writer", "tensorflow.python.ops.summary_ops_v2.always_record_summaries", "tensorflow.python.ops.summary_ops_v2.import_event", "tensorflow.python.framework.constant_op.constant"], "tensorflow.python.summary.writer.event_file_writer.EventFileWriter.__init__": ["<builtin>.str", "six.moves.queue.Queue", "tensorflow.python.util.compat.as_bytes", "os.path.join", "tensorflow.python.summary.writer.event_file_writer.EventFileWriter._get_sentinel_event", "tensorflow.python.summary.writer.event_file_writer._EventLoggerThread.__init__"], "tensorflow.python.summary.writer.writer.SummaryToEventTransformer.__init__": ["tensorflow.python.summary.writer.writer.SummaryToEventTransformer.add_graph", "tensorflow.python.summary.writer.writer.SummaryToEventTransformer.add_meta_graph", "<builtin>.isinstance", "tensorflow.python.framework.meta_graph.create_meta_graph_def", "<builtin>.set"], "tensorflow.python.ops.gen_array_ops.reshape_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.math_ops._may_reduce_to_scalar": ["tensorflow.python.framework.common_shapes.has_fully_defined_shape"], "tensorflow.python.ops.gen_math_ops._min": ["tensorflow.python.eager.execute.make_bool", "tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_math_ops._min_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.math_ops._ReductionDims": ["<builtin>.ValueError", "tensorflow.python.framework.common_shapes.rank", "tensorflow.python.ops.math_ops.range", "tensorflow.python.framework.constant_op.constant", "<builtin>.isinstance", "tensorflow.python.ops.array_ops.rank", "numpy.arange"], "tensorflow.python.ops.random_ops._ShapeTensor": ["<builtin>.isinstance", "tensorflow.python.framework.ops.convert_to_tensor"], "tensorflow.python.framework.random_seed.get_seed": ["tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.eager.context.global_seed", "tensorflow.python.framework.ops.get_default_graph", "tensorflow.python.framework.random_seed._truncate_seed", "tensorflow.python.eager.context.internal_operation_seed"], "tensorflow.python.ops.gen_random_ops.random_standard_normal": ["tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.make_int", "tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_random_ops.random_standard_normal_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.gen_nn_ops.conv2d_eager_fallback": ["tensorflow.python.eager.context.context", "<builtin>.isinstance", "<builtin>.TypeError", "tensorflow.python.eager.execute.make_int", "tensorflow.python.eager.execute.make_str", "tensorflow.python.eager.execute.make_bool", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.contrib.layers.python.layers.initializers.variance_scaling_initializer": ["<builtin>.TypeError", "<builtin>.float", "math.sqrt", "tensorflow.python.ops.random_ops.random_uniform", "tensorflow.python.ops.random_ops.truncated_normal", "<builtin>.len"], "tensorflow.python.ops.gen_math_ops.mean": ["tensorflow.python.eager.execute.make_bool", "tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_math_ops.mean_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.util.compat.as_bytes": ["<builtin>.isinstance", "<builtin>.TypeError"], "tensorflow.python.eager.execute.args_to_matching_eager": ["<builtin>.isinstance"], "tensorflow.python.framework.ops.get_collection": ["tensorflow.python.framework.ops.get_default_graph"], "tensorflow.python.framework.ops.add_to_collection": ["tensorflow.python.framework.ops.get_default_graph"], "tensorflow.python.ops.variable_scope.get_variable_scope_store": ["tensorflow.python.framework.ops.get_collection", "tensorflow.python.ops.variable_scope._VariableScopeStore.__init__", "tensorflow.python.framework.ops.add_to_collection"], "tensorflow.python.framework.ops.internal_convert_to_tensor": ["tensorflow.python.eager.context.context", "<builtin>.isinstance", "tensorflow.python.framework.ops.get_default_graph", "<builtin>.RuntimeError", "tensorflow.python.framework.dtypes.as_dtype", "<builtin>.type", "<builtin>.sorted", "<builtin>.TypeError", "tensorflow.python.framework.ops._error_prefix"], "tensorflow.python.ops.array_ops.zeros": ["tensorflow.python.framework.dtypes.as_dtype", "tensorflow.python.framework.ops.name_scope.__init__", "<builtin>.isinstance", "tensorflow.python.framework.constant_op.constant", "tensorflow.python.ops.array_ops._constant_if_small", "tensorflow.python.framework.constant_op._tensor_shape_tensor_conversion_function", "tensorflow.python.framework.tensor_shape.TensorShape.__init__", "tensorflow.python.framework.ops.convert_to_tensor"], "tensorflow.python.ops.gen_array_ops.zeros_like": ["tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_array_ops.zeros_like_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.framework.ops.device": ["tensorflow.python.eager.context.executing_eagerly", "<builtin>.callable", "tensorflow.python.eager.context.device", "<builtin>.RuntimeError", "tensorflow.python.framework.ops.get_default_graph"], "tensorflow.python.ops.array_ops.shape_internal": ["tensorflow.python.framework.ops.name_scope.__init__", "<builtin>.isinstance", "tensorflow.python.ops.gen_math_ops.cast", "tensorflow.python.ops.gen_array_ops.shape", "tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.framework.constant_op.constant"], "tensorflow.python.ops.gen_array_ops.where": ["tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_array_ops.where_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.gen_math_ops.select": ["tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_math_ops.select_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.eager.context._initialize_context": ["tensorflow.python.eager.context.Context.__init__"], "tensorflow.python.framework.ops.get_default_graph": ["tensorflow.python.framework.ops._DefaultGraphStack.get_default"], "tensorflow.python.training.saver.Saver._build": ["tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.training.saver.BaseSaverBuilder._build_internal", "tensorflow.python.training.saver.BulkSaverBuilder.__default_init__", "tensorflow.python.ops.variables._all_saveable_objects", "tensorflow.python.framework.ops.prepend_name_scope", "<builtin>.ValueError", "tensorflow.python.training.saver.Saver._check_saver_def", "time.time"], "tensorflow.python.eager.execute.make_shape": ["tensorflow.python.framework.tensor_shape.as_shape", "<builtin>.TypeError", "<builtin>.ValueError"], "tensorflow.python.ops.gen_array_ops.placeholder_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.make_shape", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_math_ops.arg_max_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.framework.dtypes.as_dtype": ["<builtin>.isinstance", "<builtin>.TypeError"], "tensorflow.python.ops.array_ops.shape": ["tensorflow.python.ops.array_ops.shape_internal"], "tensorflow.python.framework.tensor_shape.as_shape": ["<builtin>.isinstance", "tensorflow.python.framework.tensor_shape.TensorShape.__init__"], "tensorflow.python.framework.tensor_shape.TensorShape.__init__": ["<builtin>.isinstance", "<builtin>.TypeError", "tensorflow.python.framework.tensor_shape.as_dimension", "<builtin>.iter"], "tensorflow.python.ops.gen_random_ops.random_uniform_int": ["tensorflow.python.eager.execute.make_int", "tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_random_ops.random_uniform_int_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.gen_random_ops.random_uniform": ["tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.make_int", "tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_random_ops.random_uniform_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.math_ops._div_python2": ["tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.framework.ops.convert_to_tensor", "<builtin>.TypeError", "tensorflow.python.ops.gen_math_ops.real_div", "tensorflow.python.ops.gen_math_ops.floor_div"], "tensorflow.python.framework.tensor_util._ConstantValue": ["<builtin>.isinstance", "<builtin>.TypeError", "tensorflow.python.framework.tensor_util.MakeNdarray", "numpy.array", "numpy.prod", "numpy.ndarray", "tensorflow.python.framework.tensor_util.constant_value", "numpy.arange", "tensorflow.python.framework.dtypes.as_dtype", "numpy.concatenate", "numpy.full", "numpy.equal", "numpy.not_equal"], "tensorflow.python.ops.control_flow_ops.group": ["tensorflow.python.eager.context.executing_eagerly", "<builtin>.ValueError", "tensorflow.python.framework.ops.name_scope.__init__", "<builtin>.sorted", "<builtin>.len", "tensorflow.python.ops.control_flow_ops._GroupControlDeps", "six.iterkeys", "tensorflow.python.framework.ops.control_dependencies", "<builtin>.hasattr", "<builtin>.TypeError", "<builtin>.type"], "tensorflow.python.ops.gen_math_ops.batch_mat_mul_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_bool", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_math_ops.conj": ["tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_math_ops.conj_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.gen_math_ops.mat_mul_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_bool", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.framework.sparse_tensor.SparseTensor.__init__": ["tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.framework.ops.internal_convert_to_tensor"], "tensorflow.python.platform.tf_logging.warn": ["tensorflow.python.platform.tf_logging._get_logger"], "tensorflow.python.framework.ops.IndexedSlices.__init__": ["tensorflow.python.framework.ops._get_graph_from_inputs"], "tensorflow.python.ops.summary_ops_v2.create_file_writer": ["tensorflow.python.ops.summary_ops_v2.SummaryWriter.__init__", "<builtin>.str", "tensorflow.python.framework.ops.device", "tensorflow.python.ops.summary_ops_v2._make_summary_writer", "tensorflow.python.framework.constant_op.constant"], "tensorflow.python.ops.summary_ops_v2.always_record_summaries": ["tensorflow.python.framework.ops.get_collection_ref"], "tensorflow.python.ops.summary_ops_v2.import_event": ["tensorflow.python.ops.gen_summary_ops.import_event", "tensorflow.python.eager.context.context"], "tensorflow.python.framework.constant_op.constant": ["tensorflow.python.eager.context.context", "tensorflow.python.framework.constant_op.convert_to_eager_tensor", "tensorflow.python.framework.tensor_shape.as_shape", "<builtin>.TypeError", "tensorflow.python.framework.constant_op._eager_reshape", "tensorflow.python.framework.constant_op._eager_identity", "tensorflow.python.framework.constant_op._eager_fill", "tensorflow.python.framework.ops.device", "<builtin>.tuple", "tensorflow.python.framework.ops.get_default_graph", "tensorflow.python.framework.tensor_util.make_tensor_proto"], "tensorflow.python.summary.writer.event_file_writer._EventLoggerThread.__init__": ["threading.Thread.__init__"], "tensorflow.python.summary.writer.writer.SummaryToEventTransformer.add_graph": ["<builtin>.ValueError", "<builtin>.isinstance", "tensorflow.python.summary.writer.writer.SummaryToEventTransformer._write_plugin_assets", "tensorflow.python.platform.tf_logging.warning", "<builtin>.TypeError", "tensorflow.python.summary.writer.writer.SummaryToEventTransformer._add_graph_def"], "tensorflow.python.summary.writer.writer.SummaryToEventTransformer.add_meta_graph": ["<builtin>.isinstance", "<builtin>.TypeError", "<builtin>.type", "tensorflow.python.summary.writer.writer.SummaryToEventTransformer._add_event"], "tensorflow.python.framework.meta_graph.create_meta_graph_def": ["<builtin>.TypeError", "<builtin>.isinstance", "<builtin>.type", "tensorflow.python.framework.ops.get_default_graph", "<builtin>.len", "tensorflow.python.framework.meta_graph.stripped_op_list_for_graph", "tensorflow.python.framework.meta_graph._strip_graph_default_valued_attrs", "tensorflow.python.framework.ops.get_from_proto_function", "tensorflow.python.framework.meta_graph.add_collection_def"], "tensorflow.python.framework.common_shapes.has_fully_defined_shape": ["<builtin>.isinstance"], "tensorflow.python.ops.gen_math_ops._min_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_bool", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.framework.common_shapes.rank": ["<builtin>.isinstance"], "tensorflow.python.ops.math_ops.range": ["tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.ops.gen_math_ops._range", "<builtin>.all", "<builtin>.max", "tensorflow.python.ops.math_ops.cast"], "tensorflow.python.ops.array_ops.rank": ["tensorflow.python.ops.array_ops.rank_internal"], "tensorflow.python.eager.context.global_seed": ["tensorflow.python.eager.context.context"], "tensorflow.python.eager.context.internal_operation_seed": ["tensorflow.python.eager.context.context"], "tensorflow.python.ops.gen_random_ops.random_standard_normal_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.make_int", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.random_ops.truncated_normal": ["tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.ops.random_ops._ShapeTensor", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.framework.random_seed.get_seed", "tensorflow.python.ops.gen_random_ops.truncated_normal"], "tensorflow.python.ops.gen_math_ops.mean_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_bool", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.variable_scope._VariableScopeStore.__init__": ["<builtin>.super", "tensorflow.python.ops.variable_scope.VariableScope.__init__"], "tensorflow.python.ops.array_ops._constant_if_small": ["numpy.prod", "tensorflow.python.framework.constant_op.constant"], "tensorflow.python.framework.constant_op._tensor_shape_tensor_conversion_function": ["<builtin>.ValueError", "<builtin>.TypeError", "tensorflow.python.framework.constant_op.constant"], "tensorflow.python.ops.gen_array_ops.zeros_like_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.eager.context.device": ["tensorflow.python.eager.context.context"], "tensorflow.python.ops.gen_array_ops.shape": ["tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_array_ops.shape_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.gen_array_ops.where_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_math_ops.select_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.eager.context.Context.__init__": ["tensorflow.python.eager.context._EagerContext.__init__", "tensorflow.python.eager.context._ContextSwitchStack.__init__", "tensorflow.python.eager.context.Context.executing_eagerly", "threading.Lock", "<builtin>.ValueError"], "tensorflow.python.framework.ops._DefaultGraphStack.get_default": ["tensorflow.python.framework.ops._DefaultStack.get_default", "<builtin>.super", "tensorflow.python.framework.ops._DefaultGraphStack._GetGlobalDefaultGraph"], "tensorflow.python.training.saver.BaseSaverBuilder._build_internal": ["<builtin>.ValueError", "tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.training.saver.BaseSaverBuilder._ValidateAndSliceInputs", "tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.framework.constant_op.constant", "tensorflow.python.training.saver.BaseSaverBuilder._GroupByDevices", "tensorflow.python.training.saver.BaseSaverBuilder._AddShardedSaveOps", "tensorflow.python.training.saver.BaseSaverBuilder._AddShardedRestoreOps", "tensorflow.python.training.saver.BaseSaverBuilder._AddSaveOps", "tensorflow.python.training.saver.BaseSaverBuilder._AddRestoreOps", "tensorflow.python.framework.ops.get_default_graph", "<builtin>.isinstance"], "tensorflow.python.training.saver.BulkSaverBuilder.__default_init__": ["tensorflow.python.training.saver.BaseSaverBuilder.__init__"], "tensorflow.python.ops.variables._all_saveable_objects": ["tensorflow.python.framework.ops.get_collection"], "tensorflow.python.framework.ops.prepend_name_scope": ["re.sub", "tensorflow.python.platform.tf_logging.warning"], "tensorflow.python.framework.tensor_shape.as_dimension": ["<builtin>.isinstance", "tensorflow.python.framework.tensor_shape.Dimension.__init__"], "tensorflow.python.ops.gen_random_ops.random_uniform_int_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_int", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_random_ops.random_uniform_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.make_int", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_math_ops.real_div": ["tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_math_ops.real_div_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.gen_math_ops.floor_div": ["tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_math_ops.floor_div_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.framework.tensor_util.MakeNdarray": ["numpy.prod", "tensorflow.python.framework.dtypes.as_dtype", "<builtin>.len", "numpy.array", "numpy.fromiter", "numpy.frombuffer", "numpy.repeat", "<builtin>.iter", "<builtin>.TypeError", "<builtin>.complex", "<builtin>.zip"], "tensorflow.python.ops.control_flow_ops._GroupControlDeps": ["tensorflow.python.framework.ops.control_dependencies", "tensorflow.python.framework.ops.device"], "tensorflow.python.framework.ops.control_dependencies": ["tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.framework.ops._NullContextmanager.__default_init__", "<builtin>.callable", "tensorflow.python.framework.ops.get_default_graph"], "tensorflow.python.ops.gen_math_ops.conj_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.platform.tf_logging._get_logger": ["logging.getLogger", "logging.StreamHandler", "logging.Formatter"], "tensorflow.python.framework.ops._get_graph_from_inputs": ["tensorflow.python.framework.ops.get_default_graph", "<builtin>.tuple", "<builtin>.TypeError", "<builtin>.isinstance", "tensorflow.python.framework.ops._as_graph_element", "<builtin>.type", "tensorflow.python.framework.ops._assert_same_graph", "<builtin>.ValueError"], "tensorflow.python.ops.summary_ops_v2.SummaryWriter.__init__": ["tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.ops.resource_variable_ops.EagerResourceDeleter.__init__"], "tensorflow.python.ops.summary_ops_v2._make_summary_writer": ["tensorflow.python.ops.gen_summary_ops.summary_writer", "tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.framework.ops.add_to_collection", "tensorflow.python.ops.summary_ops_v2.SummaryWriter.__init__"], "tensorflow.python.framework.ops.get_collection_ref": ["tensorflow.python.framework.ops.get_default_graph"], "tensorflow.python.ops.gen_summary_ops.import_event": ["tensorflow.python.ops.gen_summary_ops.import_event_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.framework.constant_op.convert_to_eager_tensor": ["<builtin>.isinstance", "<builtin>.TypeError", "tensorflow.python.framework.dtypes.as_dtype", "<builtin>.type"], "tensorflow.python.framework.constant_op._eager_reshape": ["tensorflow.python.eager.execute.args_to_matching_eager"], "tensorflow.python.framework.constant_op._eager_fill": ["tensorflow.python.framework.constant_op.convert_to_eager_tensor"], "tensorflow.python.framework.tensor_util.make_tensor_proto": ["<builtin>.isinstance", "tensorflow.python.framework.dtypes.as_dtype", "<builtin>.callable", "numpy.asarray", "<builtin>.getattr", "<builtin>.ValueError", "numpy.empty", "tensorflow.python.framework.tensor_util._AssertCompatible", "numpy.array", "numpy.prod", "numpy.array_equal", "<builtin>.list", "tensorflow.python.framework.tensor_util._GetDenseDimensions", "<builtin>.TypeError", "<builtin>.hasattr", "<builtin>.int", "<builtin>.tuple", "tensorflow.python.framework.tensor_shape.as_shape", "tensorflow.python.framework.tensor_util._FlattenToStrings", "tensorflow.python.util.compat.as_bytes", "<builtin>.type", "tensorflow.python.framework.tensor_util.GetNumpyAppendFn"], "tensorflow.python.summary.writer.writer.SummaryToEventTransformer._write_plugin_assets": ["tensorflow.python.summary.plugin_asset.get_all_plugin_assets", "os.path.join"], "tensorflow.python.platform.tf_logging.warning": ["tensorflow.python.platform.tf_logging._get_logger"], "tensorflow.python.summary.writer.writer.SummaryToEventTransformer._add_graph_def": ["tensorflow.python.summary.writer.writer.SummaryToEventTransformer._add_event"], "tensorflow.python.summary.writer.writer.SummaryToEventTransformer._add_event": ["time.time", "<builtin>.int"], "tensorflow.python.framework.meta_graph.stripped_op_list_for_graph": ["tensorflow.python.framework.meta_graph.ops_used_by_graph_def", "tensorflow.python.framework.op_def_registry.get_registered_ops", "<builtin>.ValueError", "<builtin>.sorted"], "tensorflow.python.framework.meta_graph._strip_graph_default_valued_attrs": ["tensorflow.python.framework.op_def_registry.get_registered_ops", "<builtin>.set", "tensorflow.python.framework.meta_graph._is_default_attr_value"], "tensorflow.python.framework.ops.get_from_proto_function": ["tensorflow.python.framework.registry.Registry.lookup"], "tensorflow.python.framework.meta_graph.add_collection_def": ["<builtin>.TypeError", "<builtin>.isinstance", "<builtin>.type", "tensorflow.python.platform.tf_logging.warning", "tensorflow.python.framework.ops.get_default_graph", "tensorflow.python.framework.meta_graph._should_include_node", "tensorflow.python.framework.ops.get_to_proto_function", "tensorflow.python.framework.ops.get_collection_proto_type", "tensorflow.python.framework.meta_graph._get_kind_name", "<builtin>.str", "tensorflow.python.framework.ops.strip_name_scope", "tensorflow.python.util.compat.as_bytes", "<builtin>.getattr"], "tensorflow.python.ops.gen_math_ops._range": ["tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_math_ops._range_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.array_ops.rank_internal": ["tensorflow.python.framework.ops.name_scope.__init__", "<builtin>.isinstance", "tensorflow.python.ops.gen_array_ops.size", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.ops.gen_array_ops.rank", "tensorflow.python.framework.constant_op.constant"], "tensorflow.python.ops.gen_random_ops.truncated_normal": ["tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.make_int", "tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_random_ops.truncated_normal_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.variable_scope.VariableScope.__init__": ["tensorflow.python.eager.context.executing_eagerly", "<builtin>.NotImplementedError"], "tensorflow.python.ops.gen_array_ops.shape_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.eager.context._EagerContext.__init__": ["<builtin>.super", "tensorflow.python.eager.context._EagerTensorCache.__init__"], "tensorflow.python.eager.context._ContextSwitchStack.__init__": ["<builtin>.super", "tensorflow.python.eager.context._ContextSwitchStack.push"], "tensorflow.python.framework.ops._DefaultStack.get_default": ["<builtin>.len"], "tensorflow.python.framework.ops._DefaultGraphStack._GetGlobalDefaultGraph": ["tensorflow.python.framework.ops.Graph.__init__"], "tensorflow.python.training.saver.BaseSaverBuilder._ValidateAndSliceInputs": ["<builtin>.isinstance", "tensorflow.python.training.saver.BaseSaverBuilder.OpListToDict", "<builtin>.set", "<builtin>.sorted", "tensorflow.python.training.saver.BaseSaverBuilder.SaveableObjectsForOp", "tensorflow.python.training.saver.BaseSaverBuilder._AddSaveable"], "tensorflow.python.training.saver.BaseSaverBuilder._GroupByDevices": ["collections.defaultdict", "<builtin>.set", "<builtin>.len", "<builtin>.ValueError", "tensorflow.python.framework.device.canonical_name", "<builtin>.sorted"], "tensorflow.python.training.saver.BaseSaverBuilder._AddShardedSaveOps": ["tensorflow.python.training.saver.BaseSaverBuilder._AddShardedSaveOpsForV2", "<builtin>.len", "tensorflow.python.framework.constant_op.constant", "<builtin>.enumerate", "tensorflow.python.framework.ops.device", "tensorflow.python.training.saver.BaseSaverBuilder.sharded_filename", "tensorflow.python.training.saver.BaseSaverBuilder._AddSaveOps", "tensorflow.python.framework.ops.control_dependencies", "tensorflow.python.ops.gen_io_ops.sharded_filespec"], "tensorflow.python.training.saver.BaseSaverBuilder._AddShardedRestoreOps": ["<builtin>.enumerate", "tensorflow.python.framework.ops.device", "tensorflow.python.training.saver.BaseSaverBuilder._AddRestoreOps", "tensorflow.python.ops.control_flow_ops.group"], "tensorflow.python.training.saver.BaseSaverBuilder._AddSaveOps": ["tensorflow.python.training.saver.BaseSaverBuilder.save_op", "tensorflow.python.ops.control_flow_ops.with_dependencies"], "tensorflow.python.training.saver.BaseSaverBuilder._AddRestoreOps": ["tensorflow.python.training.saver.BaseSaverBuilder.bulk_restore", "<builtin>.len", "tensorflow.python.ops.array_ops.shape", "tensorflow.python.ops.control_flow_ops.group"], "tensorflow.python.framework.tensor_shape.Dimension.__init__": ["<builtin>.isinstance", "<builtin>.TypeError", "<builtin>.int", "<builtin>.ValueError"], "tensorflow.python.ops.gen_math_ops.real_div_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_math_ops.floor_div_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.framework.ops._as_graph_element": ["<builtin>.getattr", "<builtin>.callable"], "tensorflow.python.framework.ops._assert_same_graph": ["<builtin>.ValueError"], "tensorflow.python.ops.resource_variable_ops.EagerResourceDeleter.__init__": ["<builtin>.isinstance", "<builtin>.ValueError"], "tensorflow.python.ops.gen_summary_ops.summary_writer": ["tensorflow.python.eager.execute.make_str", "tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_summary_ops.summary_writer_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.gen_summary_ops.import_event_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.framework.ops.convert_to_tensor"], "tensorflow.python.framework.tensor_util._AssertCompatible": ["tensorflow.python.framework.tensor_util._FirstNotNone", "<builtin>.TypeError", "<builtin>.repr", "<builtin>.type"], "tensorflow.python.framework.tensor_util._GetDenseDimensions": ["<builtin>.isinstance", "tensorflow.python.framework.tensor_util._GetDenseDimensions", "<builtin>.len"], "tensorflow.python.framework.tensor_util._FlattenToStrings": ["<builtin>.isinstance", "tensorflow.python.framework.tensor_util._FlattenToStrings"], "tensorflow.python.framework.tensor_util.GetNumpyAppendFn": ["tensorflow.python.framework.tensor_util.GetFromNumpyDTypeDict"], "tensorflow.python.summary.plugin_asset.get_all_plugin_assets": ["tensorflow.python.framework.ops.get_default_graph", "<builtin>.len", "<builtin>.ValueError"], "tensorflow.python.framework.meta_graph.ops_used_by_graph_def": ["<builtin>.set"], "tensorflow.python.framework.registry.Registry.lookup": ["<builtin>.LookupError"], "tensorflow.python.framework.meta_graph._should_include_node": ["<builtin>.isinstance"], "tensorflow.python.framework.ops.get_to_proto_function": ["tensorflow.python.framework.registry.Registry.lookup"], "tensorflow.python.framework.ops.get_collection_proto_type": ["tensorflow.python.framework.registry.Registry.lookup"], "tensorflow.python.framework.meta_graph._get_kind_name": ["<builtin>.isinstance"], "tensorflow.python.framework.ops.strip_name_scope": ["re.sub", "tensorflow.python.platform.tf_logging.warning"], "tensorflow.python.ops.gen_math_ops._range_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_array_ops.size": ["tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_array_ops.size_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.gen_array_ops.rank": ["tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_array_ops.rank_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.gen_random_ops.truncated_normal_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.make_int", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.eager.context._EagerTensorCache.__init__": ["collections.OrderedDict"], "tensorflow.python.training.saver.BaseSaverBuilder.OpListToDict": ["<builtin>.isinstance", "<builtin>.TypeError", "<builtin>.sorted", "<builtin>.ValueError", "tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.training.saver.BaseSaverBuilder.OpListToDict", "<builtin>.callable", "tensorflow.python.framework.ops.internal_convert_to_tensor", "tensorflow.python.training.saver.BaseSaverBuilder._IsVariable", "<builtin>.type"], "tensorflow.python.training.saver.BaseSaverBuilder.SaveableObjectsForOp": ["<builtin>.isinstance", "<builtin>.TypeError", "<builtin>.list", "tensorflow.python.eager.context.executing_eagerly", "<builtin>.ValueError", "tensorflow.python.training.saver.BaseSaverBuilder.SaveableObjectsForOp", "tensorflow.python.training.saver.BaseSaverBuilder.VariableSaveable.__init__", "tensorflow.python.training.saver.BaseSaverBuilder.ResourceVariableSaveable.__init__", "<builtin>.callable", "tensorflow.python.framework.ops.internal_convert_to_tensor", "tensorflow.python.training.saver.BaseSaverBuilder._IsVariable", "<builtin>.type"], "tensorflow.python.training.saver.BaseSaverBuilder._AddSaveable": ["<builtin>.ValueError"], "tensorflow.python.framework.device.canonical_name": ["<builtin>.isinstance", "tensorflow.python.framework.device.DeviceSpec.from_string"], "tensorflow.python.training.saver.BaseSaverBuilder._AddShardedSaveOpsForV2": ["uuid.uuid4", "<builtin>.len", "tensorflow.python.framework.constant_op.constant", "<builtin>.enumerate", "tensorflow.python.framework.ops.device", "tensorflow.python.training.saver.BaseSaverBuilder.sharded_filename", "tensorflow.python.training.saver._set_cpu0", "tensorflow.python.training.saver.BaseSaverBuilder._AddSaveOps", "tensorflow.python.framework.ops.control_dependencies", "tensorflow.python.ops.gen_io_ops.merge_v2_checkpoints", "tensorflow.python.ops.array_ops.identity"], "tensorflow.python.training.saver.BaseSaverBuilder.sharded_filename": ["tensorflow.python.ops.gen_io_ops.sharded_filename"], "tensorflow.python.ops.gen_io_ops.sharded_filespec": ["tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_io_ops.sharded_filespec_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.training.saver.BaseSaverBuilder.save_op": ["tensorflow.python.ops.io_ops._save", "<builtin>.RuntimeError"], "tensorflow.python.ops.control_flow_ops.with_dependencies": ["tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.framework.ops.colocate_with", "<builtin>.list", "tensorflow.python.framework.ops.control_dependencies", "tensorflow.python.framework.ops.convert_to_tensor_or_indexed_slices", "<builtin>.isinstance", "tensorflow.python.ops.control_flow_ops._Identity", "tensorflow.python.framework.ops.IndexedSlices.__init__"], "tensorflow.python.training.saver.BaseSaverBuilder.bulk_restore": ["tensorflow.python.framework.ops.device", "tensorflow.python.training.saver.BaseSaverBuilder.restore_op", "tensorflow.python.training.saver._set_cpu0"], "tensorflow.python.ops.gen_summary_ops.summary_writer_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_str", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.framework.tensor_util._FirstNotNone": ["<builtin>.isinstance", "tensorflow.python.framework.tensor_util._Message.__init__"], "tensorflow.python.framework.tensor_util.GetFromNumpyDTypeDict": ["six.iteritems"], "tensorflow.python.ops.gen_array_ops.size_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_array_ops.rank_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.training.saver.BaseSaverBuilder._IsVariable": ["<builtin>.isinstance"], "tensorflow.python.training.saver.BaseSaverBuilder.VariableSaveable.__init__": ["<builtin>.super"], "tensorflow.python.training.saver.BaseSaverBuilder.ResourceVariableSaveable.__init__": ["<builtin>.isinstance", "<builtin>.ValueError", "<builtin>.repr", "tensorflow.python.framework.ops.device", "tensorflow.python.ops.array_ops.identity", "<builtin>.super"], "tensorflow.python.framework.device.DeviceSpec.from_string": ["tensorflow.python.framework.device.DeviceSpec.__init__"], "tensorflow.python.ops.gen_io_ops.merge_v2_checkpoints": ["tensorflow.python.eager.execute.make_bool", "tensorflow.python.ops.gen_io_ops.merge_v2_checkpoints_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.array_ops.identity": ["tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.ops.gen_array_ops.identity", "tensorflow.python.eager.context.context"], "tensorflow.python.ops.gen_io_ops.sharded_filename": ["tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_io_ops.sharded_filename_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.gen_io_ops.sharded_filespec_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.io_ops._save": ["tensorflow.python.ops.gen_io_ops.save", "tensorflow.python.ops.gen_io_ops.save_slices"], "tensorflow.python.framework.ops.colocate_with": ["tensorflow.python.framework.ops._colocate_with_for_gradient"], "tensorflow.python.framework.ops.convert_to_tensor_or_indexed_slices": ["tensorflow.python.framework.ops.internal_convert_to_tensor_or_indexed_slices"], "tensorflow.python.ops.control_flow_ops._Identity": ["tensorflow.python.framework.ops.internal_convert_to_tensor_or_indexed_slices", "<builtin>.isinstance", "tensorflow.python.ops.control_flow_ops._Identity", "tensorflow.python.ops.array_ops.identity", "tensorflow.python.ops.gen_array_ops.ref_identity", "<builtin>.TypeError", "tensorflow.python.framework.ops.IndexedSlices.__init__", "tensorflow.python.framework.sparse_tensor.SparseTensor.__init__", "<builtin>.type"], "tensorflow.python.framework.device.DeviceSpec.__init__": ["<builtin>.hash", "tensorflow.python.framework.device.DeviceSpec.to_string"], "tensorflow.python.ops.gen_io_ops.merge_v2_checkpoints_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_bool", "tensorflow.python.framework.ops.convert_to_tensor"], "tensorflow.python.ops.gen_array_ops.identity": ["tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_array_ops.identity_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.gen_io_ops.sharded_filename_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_io_ops.save": ["tensorflow.python.ops.gen_io_ops.save_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.gen_io_ops.save_slices": ["tensorflow.python.ops.gen_io_ops.save_slices_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.framework.ops._colocate_with_for_gradient": ["tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.framework.ops.get_default_graph", "<builtin>.isinstance", "tensorflow.python.framework.ops.device", "tensorflow.python.framework.ops._NullContextmanager.__default_init__", "<builtin>.ValueError"], "tensorflow.python.framework.ops.internal_convert_to_tensor_or_indexed_slices": ["<builtin>.isinstance", "tensorflow.python.framework.ops.internal_convert_to_tensor", "tensorflow.python.eager.context.executing_eagerly", "<builtin>.ValueError", "tensorflow.python.framework.dtypes.as_dtype", "<builtin>.str"], "tensorflow.python.ops.gen_array_ops.ref_identity": ["tensorflow.python.eager.execute.record_gradient", "<builtin>.RuntimeError"], "tensorflow.python.framework.device.DeviceSpec.to_string": ["<builtin>.str"], "tensorflow.python.ops.gen_array_ops.identity_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_io_ops.save_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.convert_to_mixed_eager_tensors", "tensorflow.python.framework.ops.convert_to_tensor", "<builtin>.list"], "tensorflow.python.ops.gen_io_ops.save_slices_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.convert_to_mixed_eager_tensors", "tensorflow.python.framework.ops.convert_to_tensor", "<builtin>.list"], "tensorflow.python.eager.execute.convert_to_mixed_eager_tensors": ["tensorflow.python.framework.ops.internal_convert_to_tensor"]}